src/doc/frames_sequence.h

```diff
@@ -0,0 +1,70 @@
+// Aseprite Document Library
+// Copyright (c) 2023 Igara Studio S.A.
+//
+// This file is released under the terms of the MIT license.
+// Read LICENSE.txt for more information.
+
+#ifndef DOC_FRAMES_SEQUENCE_H_INCLUDED
+#define DOC_FRAMES_SEQUENCE_H_INCLUDED
+#pragma once
+
+#include "doc/frame_range.h"
+#include "doc/selected_frames.h"
+
+#include <iosfwd>
+#include <vector>
+
+namespace doc {
+
+  using namespace frames;
+
+  class FramesSequence {
```

Comment:
Add a comment that this class is based in `SelectedFrames` and a `TODO` to for future work to be done to remove all the duplicated code in some way. A similar comment can go to the `SelectedFrames` class definition.